bin/validate/atomic-red-team.schema.yaml

```diff
@@ -21,6 +21,7 @@ properties:
 $defs:
   test:
     type: object
+    additionalProperties: false
```

Comment:
Validation checks fail if there are any additional properties specified apart from the ones that we described.